src/Pos/Ssc/GodTossing/Arbitrary.hs

```diff
@@ -39,12 +42,46 @@ import           Pos.Util.Arbitrary               (Nonrepeating (..), makeSmall,
 -- Core
 ----------------------------------------------------------------------------
 
+-- | Wrapper over 'Commitment'. Creates an invalid Commitment w.r.t. 'verifyCommitment'.
+newtype BadCommitment = BadComm
+    { getBadComm :: Commitment
+    } deriving (Show, Eq)
+
+instance Arbitrary BadCommitment where
+    arbitrary = BadComm <$> do
+      Commitment <$> arbitrary <*> arbitrary <*> (arbitrary `suchThat` (not . null))
```

Comment:
Oh, I forgot to remove this part...
It's related to the edge case I mentioned about `verifyCommitment`. I'll remove this now.